packages/tasks/src/sendZETA.ts

```diff
@@ -0,0 +1,69 @@
+import confirm from "@inquirer/confirm";
+import { task } from "hardhat/config";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { ZetaChainClient } from "../../client/src/";
+
+declare const hre: any;
```

Comment:
What's the meaning of this? Can we remove it?

Reply:
720e529